net/mwan3/Makefile

```diff
@@ -8,9 +8,9 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mwan3
-PKG_VERSION:=2.0
-PKG_RELEASE:=3
-PKG_MAINTAINER:=Jeroen Louwes <jeroen.louwes@gmail.com>
+PKG_VERSION:=2.0.1
+PKG_RELEASE:=4
```

Comment:
@champtar I know but the source are in your repo so the PKG_RELEASE would always be set to '1' the only exemption would be if i update the make file and not the sources.
I have talked to @rmilecki and the commits are bugfixes scattered over different version in the master. He agreed that i should use a version which only fits to this branch.